python/pyspark/sql/dataframe.py

```diff
@@ -351,8 +352,62 @@ def show(self, n=20, truncate=True, vertical=False):
         else:
             print(self._jdf.showString(n, int(truncate), vertical))
 
+    def _get_repl_config(self):
+        """Return the configs for eager evaluation each time when __repr__ or
+        _repr_html_ called by user or notebook.
+        """
+        eager_eval = self.sql_ctx.getConf(
+            "spark.sql.repl.eagerEval.enabled", "false").lower() == "true"
+        console_row = int(self.sql_ctx.getConf(
+            "spark.sql.repl.eagerEval.maxNumRows", u"20"))
```

Comment:
Do we need `u` here?

Reply:
Just follow the doc in https://github.com/apache/spark/blob/master/python/pyspark/sql/context.py#L134, but finally we cast it to int, so unicode is useless? I remove it in next commit.